ui/src/index.html

```diff
@@ -1,139 +1,191 @@
+    <title>Humeur du mois 2018</title>
 
-<head>
-  <meta charset="utf-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Humeur du mois 2018</title>
+    <script
+      defer
+      src="https://cdn.auth0.com/js/auth0/9.7.3/auth0.min.js"
+    ></script>
 
-  <script defer src="https://cdn.auth0.com/js/auth0/9.7.3/auth0.min.js"></script>
+    <!-- update the version number as needed -->
+    <script defer src="/__/firebase/5.3.1/firebase-app.js"></script>
+    <!-- include only the Firebase features as you need -->
+    <script defer src="/__/firebase/5.3.1/firebase-auth.js"></script>
+    <script defer src="/__/firebase/5.3.1/firebase-firestore.js"></script>
+    <!-- initialize the SDK after all desired features are loaded -->
+    <script defer src="/__/firebase/init.js"></script>
+  </head>
 
-  <!-- update the version number as needed -->
-  <script defer src="/__/firebase/5.3.1/firebase-app.js"></script>
-  <!-- include only the Firebase features as you need -->
-  <script defer src="/__/firebase/5.3.1/firebase-auth.js"></script>
-  <script defer src="/__/firebase/5.3.1/firebase-firestore.js"></script>
-  <!-- initialize the SDK after all desired features are loaded -->
-  <script defer src="/__/firebase/init.js"></script>
-</head>
-
-<body>
-  <div class="content">
-    <div id="loggingInPage" class="page">
-      <h2>Hi!</h2>
-      <h1>I'm logging you in...</h1>
-      <div class="loader">
-        <svg class="circular" viewBox="25 25 50 50">
-          <circle class="path" cx="50" cy="50" r="20" fill="none" strokeWidth="2" strokeMiterlimit="10" />
-        </svg>
+  <body>
+    <div class="content">
+      <div id="loggingInPage" class="page">
+        <h2>Hi!</h2>
+        <h1>I'm logging you in...</h1>
+        <div class="loader">
+          <svg class="circular" viewBox="25 25 50 50">
+            <circle
+              class="path"
+              cx="50"
+              cy="50"
+              r="20"
+              fill="none"
+              strokeWidth="2"
+              strokeMiterlimit="10"
+            />
+          </svg>
+        </div>
       </div>
-    </div>
-    <div id="homePage" class="page hidden">
-      <h2>Welcome,
-        <span id="userId"></span>
-      </h2>
-      <h1>How have you been at Zenika?</h1>
-      <div class="button__wrapper">
-        <button class="button button--great" type="button" id="submitGreat">
-          <span class="button__emoji">😁</span>
-          Great!
-        </button>
-        <button class="button button--no-that-great" type="button" id="submitNotThatGreat">
-          <span class="button__emoji">😐</span>
-          Not that great
-        </button>
-        <button class="button button--not-great-at-all" type="button" id="submitNotGreatAtAll">
-          <span class="button__emoji">😤</span>
-          Not great at all
-        </button>
+      <div id="homePage" class="page hidden">
+        <h2>Welcome, <span id="userId"></span></h2>
```

Comment:
a h2 before h1 ?
Could your remove the h2, and use a <p> tag with the right CSS styles ?

Reply:
I agree this is weird, i'm gonna swap those